src/components/ImageSlider/ImageSlider.js

```diff
@@ -0,0 +1,110 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { css } from '@emotion/react';
+import { Button, Icon } from '@newrelic/gatsby-theme-newrelic';
+
+const ImageSlider = ({ images, height }) => {
+  const [selectedImageIndex, setSelectedImageIndex] = useState(0);
+
+  const handleClickNext = () => {
+    const nextImageIndex = selectedImageIndex + 1;
+    setSelectedImageIndex(nextImageIndex % images.length);
+  };
+
+  const handleClickPrev = () => {
+    const prevImageIndex = selectedImageIndex - 1;
+    if (prevImageIndex < 0) {
+      setSelectedImageIndex(images.length - 1);
+    } else {
+      setSelectedImageIndex(prevImageIndex % images.length);
+    }
+  };
+
+  return (
+    <div
+      css={css`
+        position: relative;
+        margin-bottom: 2rem;
+      `}
+    >
+      {images.length > 1 && (
+        <>
+          <Button
+            onClick={handleClickPrev}
+            variant={Button.VARIANT.PLAIN}
+            css={css`
+              position: absolute;
+              top: 38%;
+              left: 0;
+              background: none;
+              color: var(--color-white);
+              border: none;
+              cursor: pointer;
+            `}
+          >
+            <Icon name="chevron-left" size="4rem" />
+          </Button>
+          <Button
+            onClick={handleClickNext}
+            variant={Button.VARIANT.PLAIN}
+            css={css`
+              position: absolute;
+              top: 38%;
+              right: 0;
+              background: none;
+              color: var(--color-white);
+              border: none;
+              cursor: pointer;
+            `}
+          >
+            <Icon name="chevron-right" size="4rem" />
+          </Button>
+        </>
+      )}
+      <a
+        href={
+          images && images.length > 0
+            ? images[selectedImageIndex]
+            : noImagePlaceholder
+        }
+        target="_blank"
+        rel="noreferrer"
+        css={css`
+          height: ${height}px;
+          width: 100%;
+          margin-right: 1rem;
+        `}
+      >
+        <img
+          src={
+            images && images.length > 0
+              ? images[selectedImageIndex]
+              : noImagePlaceholder
+          }
+          alt="placeholder-text"
+          css={css`
+            height: ${height}px;
```

Comment:
Setting the height does some weird things on mobile:
![Screen Shot 2021-06-21 at 2 54 04 PM](https://user-images.githubusercontent.com/1946433/122832686-9321d380-d2a0-11eb-8887-b4ff0c3ec63d.png)

And it also looks like it squishes images a bit (depending on the aspect ratio). I wonder if there's a way we can avoid this 🤔 
![Screen Shot 2021-06-21 at 2 54 59 PM](https://user-images.githubusercontent.com/1946433/122832741-a765d080-d2a0-11eb-882f-c640dec1f798.png)

Reply:
yeah i made the images `object-fit: contain` on mobile screens (smaller than 760px) because seeing the entire image (where you can click and see it in full resolution in another tab) seemed slightly better than seeing small portion of the image where you can't really discern what it is. 

i adjusted so it's full width and fixed height on mobile. looks okay, but TBH not sure what best option there is.

for the squishy-ness on those medium-large screens. i adjusted that in the new PR so it doesn't squish / distort so much.